### PR TITLE
Changing my subdomain from noahvocat to frenchguy.is-a.dev

### PR DESCRIPTION
**Pull request by nvct-website**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://nvct-website.github.io

## Additional info

noahvocat.json => frenchguy.json

### CODE REVIEW

File: domains/frenchguy.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A dumb kid who likes da computer",
+  "repo": "https://github.com/nvct-website/nvct-website.github.io",
+  "owner": {
+    "username": "Noahvocat (also known as NVCT, Noahvocado or Noahvocados or FrenchGuy or AverageFrenchGuy or AvrgFrnchGuy)",
+    "email": "",
+    "discord": "avrgfrnchguy"
+  },
+  "record": {
+    "CNAME": "nvct-website.github.io"
```
Comment (DEV-DIBSTER) [inline domains/frenchguy.json:10]: This site has redirection to the domain which you are requesting to delete as well.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/frenchguy.json (hunk 1 above)
Comment (nvct-website) [reply domains/frenchguy.json:10]: I just removed the domain from the website. It’s just a Github Pages 404 now.

Comment (DEV-DIBSTER) [reply domains/frenchguy.json:10]: So is there any content on the site or no?

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**